userspace/libscap/engine/modern_bpf/CMakeLists.txt

```diff
@@ -18,12 +20,18 @@ if(RESULT STREQUAL NOTFOUND)
   message(FATAL_ERROR "problem with libbpf.cmake in ${CMAKE_MODULE_PATH}")
 endif()
 
-# Directory in which the BPF skeleton will be built
-set(SKEL_DIR "${CMAKE_BINARY_DIR}/skel_dir")
-file(MAKE_DIRECTORY ${SKEL_DIR})
-
-# Build the BPF skeleton as custom target.
-add_subdirectory(${LIBSCAP_DIR}/driver/modern_bpf ${CMAKE_BINARY_DIR}/driver/modern_bpf)
+if(NOT MODERN_BPF_SKEL_DIR)
+  message(STATUS "USE_BUNDLED_MODERN_BPF: ON")
+  # Directory in which the BPF skeleton will be built
+  set(MODERN_BPF_SKEL_DIR "${CMAKE_BINARY_DIR}/skel_dir")
+  file(MAKE_DIRECTORY ${MODERN_BPF_SKEL_DIR})
+  # Build the BPF skeleton as custom target.
+  add_subdirectory(${LIBSCAP_DIR}/driver/modern_bpf ${CMAKE_BINARY_DIR}/driver/modern_bpf)
+else()
+  message(STATUS "USE_BUNDLED_MODERN_BPF: OFF")
+  set(USE_BUNDLED_MODERN_BPF OFF)
+endif()
+message(STATUS "using modern BPF skel_dir: ${MODERN_BPF_SKEL_DIR}")
```

Comment:
[nit] Do we need 3 separate status messages here? Maybe we can unify them to use this one alone or expand the other two?

Reply:
Also, the last message might not make any sense in case `USE_BUNDLED_MODERN_BPF` is set to off, since `MODERN_BPF_SKEL_DIR` is not set on that branch.

Reply:
I think for `USE_BUNDLED_MODERN_BPF` being set to off, it must be specified explicitly, e.g., as part of the CI job.
Maybe it helps to have just one message, e.g.,
`message(STATUS "USE_BUNDLED_MODERN_BPF: ${USE_BUNDLED_MODERN_BPF} using skeleton dir: ${MODERN_BPF_SKEL_DIR}")`
Wdyt?

Reply:
done :)

Reply:
Thanks!